chart/templates/_helpers.tpl

```diff
@@ -84,6 +84,20 @@ app.kubernetes.io/version: {{ default $root.Chart.Version $root.Chart.AppVersion
 helm.sh/chart: {{ include "kubecf.chart" $root }}
 {{- end }}
 
+{{- /*
+==========================================================================================
+| kubecf.imagePullSecrets (list $ $component)
```

Comment:
You don't use the `$component` argument in the template, so you could just pass in `$` (not as a list) and get rid of `$root` inside as well.

Reply:
confirmed